runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -2414,7 +2414,7 @@ chpl_bool nextMemMapEntry(void** pAddr, size_t* pSize,
     int ch;
 
     int scn_cnt;
-    scn_cnt = fscanf(f, "%lx-%lx%4s%*x%*x:%*x%*x", &lo_addr, &hi_addr, perms);
+    scn_cnt = fscanf(f, "%"SCNu64"-%"SCNu64"%4s%*x%*x:%*x%*x", &lo_addr, &hi_addr, perms);
```

Comment:
This is the right solution, but would you mind putting spaces on both sides of the `SCNu64` tokens? That would match the formatting elsewhere in the file.

Reply:
Should be done now if you want to double-check it.